src/main/java/com/kryptnostic/api/v1/search/DefaultSearchClient.java

```diff
@@ -97,15 +96,4 @@ private SearchRequest generateSearchRequest( List<String> tokens ) {
         }
         return Lists.newArrayList( tokens );
     }
-
-    @Override
-    public SearchResultResponse nextPage( SearchResultResponse response ) {
```

Comment:
Yes. I hadn't looked at what pagination was doing closely until now and it will require a re-design to make it work again using the new C++ stuff.